### PR TITLE
Steps one & two added for the atDude Tutorial

### PR DESCRIPTION
**Pull request by CurtlyCritchlow**
**- What I did**

1. Added steps one and two to the atDude tutorial.
2. Converted the old tutorials to drafts. The draft tutorials would be removed completely after I recycle its contents.

**- How I did it**
Created the markdown files and assets for steps one and two of the atDude tutorial. 
**- How to verify it**
run `npm run start` and navigate to the atDude tutorial
**- Description for the changelog**

1. Added steps one and two to the atDude tutorial.
2. Converted the old tutorials to drafts

### CODE REVIEW

File: content_dev/docs/tutorials/at-dude/1_introduction/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: codelab
+
+title: "Introduction" # Step Name
+description: | # SEO Description for this step
+  Documentation
```
Comment (JeremyTubongbanua) [inline content_dev/docs/tutorials/at-dude/1_introduction/index.md]: Small thing- please add a description for SEO purposes

File: content_dev/docs/tutorials/at-dude/2_getting_started/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+layout: codelab
+
+title: "Getting Started" # Step Name
+description: | # SEO Description for this step
+  Documentation
```
Comment (JeremyTubongbanua) [inline content_dev/docs/tutorials/at-dude/2_getting_started/index.md]: Here too

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
2 things:
1. SEO descriptions 
2. The folder name reflects the url and we like to use dashes (-) instead of underscores (_)So instead of "1_introduction" please make it "1-introduction"

Awesome work so far

File: content_dev/docs/tutorials/at-dude/1_introduction/index.md (hunk 1 above)
Comment (CurtlyCritchlow) [reply content_dev/docs/tutorials/at-dude/1_introduction/index.md]: done

File: content_dev/docs/tutorials/at-dude/2_getting_started/index.md (hunk 2 above)
Comment (CurtlyCritchlow) [reply content_dev/docs/tutorials/at-dude/2_getting_started/index.md]: done

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by TylerTrott — APPROVED** [review]

**Pull request merged**